src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java

```diff
@@ -49,12 +48,23 @@
   P_RD_2RL2B_3(ActionHandler.PRINTER); // Response driven reminder group 3 Welsh
 
   private final ActionHandler handler;
+  private final String packCode;
 
   ActionType(ActionHandler handler) {
     this.handler = handler;
+    this.packCode = this.name();
```

Comment:
Worth noting that for some action types (e.g the ad hoc fulfilment requests) the name isn't actually the packCode, for fulfilment requests the packCode is just passed through, we don't map the action type to a packCode in this service. Perhaps we should set the packCode on those to `null` here to avoid potential confusion?

Reply:
the pack code can be overloaded if necessary, where it's different from the action type name (see below). Surely that's adequate?